src/CattyTests/DataModel/UserDataContainerTest.swift

```diff
@@ -376,8 +377,8 @@ final class UserDataContainerTest: XCTestCase {
         let userList = UserList(name: "userData")
         let userVariable = UserVariable(name: "userData")
 
-        container1.addObjectVariable(userVariable, for: object)
-        container2.addObjectList(userList, for: object)
+        container1.addVariable(userVariable)
```

Comment:
This test is not of use anymore. Please remove `testIsEqualToVariablesContainerWithObjectListAndVariableHavingSameName`.